.github/workflows/cd.yaml

```diff
@@ -0,0 +1,95 @@
+name: CD
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to build Zig wheels for'
+        required: true
+        default: 'latest'
+      suffix:
+        description: >
+          Suffix to append to the version in the wheel filename, useful for dev versions and version specifiers
+        required: false
+        default: ''
+      platforms:
+        description: >
+          Comma-separated values of platforms to build wheels for
+        required: false
+        default: 'x86_64-windows,x86-windows,x86_64-macos,aarch64-macos,i386-linux,x86-linux,x86_64-linux,aarch64-linux,armv7a-linux'
+      push_to_pypi:
+        description: >
+          Whether to push the built wheels to PyPI. Can be 'true' or 'false', defaults to 'false'.
+        required: false
+        default: 'false'
+
+jobs:
+  build_wheels:
+    name: Build wheels
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b  # v4.1.4
+      - uses: actions/setup-python@82c7e631bb3cdc910f68e0081d67478d79c6982d  # v5.1.0
+        with:
+          python-version: '3.x'
+
+      - name: Install dependencies
+        run: |
+          python -m pip install .
+
+      - name: Build wheels for all platforms
+        shell: bash
+        run: |
+          platforms=${{ github.event.inputs.platforms }}
+          IFS=',' read -r -a platform_array <<< "$platforms"
+          for platform in "${platform_array[@]}"; do
+            python make_wheels.py \
+            --version ${{ github.event.inputs.version }} \
+            --suffix ${{ github.event.inputs.suffix }} \
+            --platform "$platform"
+          done
+
+      - name: Upload wheel artifacts
+        uses: actions/upload-artifact@65462800fd760344b1a7b4382951275a0abb4808  # v4.3.3
+        with:
+          name: zig-wheels
+          path: dist/*.whl
+
+  deploy_wheels:
+    name: Deploy wheels
+    needs: [build_wheels]
+    environment: pypi
+    runs-on: ubuntu-latest
+    permissions:
+      # Required by
+      # 1. OIDC to publish to PyPI, and
+      # 2. Sigstore to sign artifacts
+      id-token: write
+    if: >-
+      github.event_name == 'workflow_dispatch' &&
+      github.event.inputs.push_to_pypi == 'true' &&
+      github.repository == 'ziglang/zig-pypi'
```

Comment:
I'll move this to the context of the step that uploads to PyPI instead of the entire job, because we can have scenarios where we want to build a wheel, upload to PyPI manually (locally) using `twine`, but also have the Sigstore files associated with it as marker that the upload has been signed.

Reply:
Addressed in cf9845ecbf36def44bfaab8e2801a395842a953d.

Reply:
Moving it back to the job's context in 55bdcc99a07e38cc080bdaf2ea35ce4a3ea30685, since it will now be better to publish the wheels to PyPI from the workflow itself.